envs/example/ci-ceph_swift/group_vars/all.yml

```diff
@@ -14,6 +14,11 @@ etc_hosts:
 
 neutron:
   enable_external_interface: True
+  lbaas:
+    enabled: True
+
+ceilometer:
+  enabled: True
 
```

Comment:
Will also need to enabled aodh when enabling ceilometer

aodh:
  enabled: True